go.mod

```diff
@@ -1,75 +1,75 @@
 module github.com/cybozu-go/pod-security-admission
 
-go 1.20
+go 1.22
```

Comment:
Is this correct?
Please check: https://github.com/golang/go/issues/62278#issuecomment-1693538776

Reply:
@zoetrope 
Oh, sorry, I forget it.
Thank you !